### ISSUE TITLE
Update documentation regarding "Encryption"

### ISSUE THREAD

**Issue by nkavian** [10cella/hashids-java#70]
`Hashids.java` uses the term `Encrypt` and `Decrypt` when the website explicitly says this does not provide encryption. Can you update the documentation? It's easy enough to simply refer to it what it is actually doing `Encode` and `Decode`.

````
  /**
   * Encrypt numbers to string
   *
   * @param numbers
   *          the numbers to encrypt
   * @return the encrypt string
   */
  public String encode(long... numbers) {
````

**Comment by 0x3333** [comment]
It has been fixed on master in commit e57ed1c4f9. Where did you find these?

**Comment by nkavian** [comment]
Maybe it was the version of the library I was using at the time. Thanks, closing.
